src/test/unit_tests/test_nc_nsfs_cli.js

```diff
@@ -530,6 +531,33 @@ mocha.describe('manage_nsfs cli', function() {
             }
         });
 
+        mocha.it('cli account add - uid=0, gid!=0', async function() {
```

Comment:
I think it would be better if you add this in jest test `test_nc_nsfs_account_cli.test.js`